lib/screens/v2/widget_instance/elevator_status.ex

```diff
@@ -212,18 +212,11 @@ defmodule Screens.V2.WidgetInstance.ElevatorStatus do
     # This assumes platform-level stop IDs are always numeric strings, e.g. "70045"
     informed_platforms =
       for %{stop: stop} when is_binary(stop) <- entities,
-          match?({_n, ""}, Integer.parse(stop)),
+          match?("place-" <> _, stop),
           do: stop
 
     connecting_platform_ids =
-      stop_sequences
-      |> List.flatten()
-      |> MapSet.new()
-      |> MapSet.difference(
-        t
-        |> platform_stop_ids()
-        |> MapSet.new()
-      )
+      stop_sequences |> List.flatten() |> List.delete(parent_station_id(t))
```

Comment:
Is this removing the `parent_station_id` since it's redundant to say that the current platform connects with itself?

Reply:
That's correct. And we don't want the parent station closures to show up in the section for connecting lines.

Reply:
gotcha, that makes sense